Makefile

```diff
@@ -254,6 +254,7 @@ configure += --enable-languages=c,c++
 configure += --enable-lto
 configure += --enable-static=yes
 configure += --disable-libstdcxx-verbose
+configure += --enable-checking=all
```

Comment:
That makes sense for the gcc development, but may degrade compiler performance for the end users.

Reply:
it's just testing purpose and will be reverted